jupyter/minimal/ubi9-python-3.12/build-args/cuda.conf

```diff
@@ -1 +1 @@
-BASE_IMAGE=quay.io/opendatahub/odh-base-image-cuda-py312-ubi9:v12.6
+BASE_IMAGE=quay.io/opendatahub/odh-base-image-cuda-py312-ubi9:v12.8
```

Comment:
I will add a review comment here as i can not add it on the `build-args` dir. 
We need the updated version of rocm image on the `rocm.conf` file as well

Reply:
changes for rocm 6.4 have been implemented as well, thanks!